Documentation/io.mender.Authentication1.xml

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+
+<node>
+  <!--
+    io.mender.Authentication1:
+    @short_description: Mender Authentication API v1
+
+    This interface lets applications authenticate with the Mender server. The
+    Mender client will handle the authentication and provide to the user with a
+    JSON Web Token (JWT) and the server URL, which which the user can do its own
+    API calls to the Mender server.
```

Comment:
This interface lets applications authenticate with the Mender server. The
Mender client will handle the authentication and provide the user with a
JSON Web Token (JWT) and the server URL, which the user can use to
API calls to the Mender server on his own.